frontend/src/views/supp-allowances/SupplementaryFormView.vue

```diff
@@ -224,6 +237,7 @@ export default {
       }
       this.clonedModels = cloneDeep(this.models)
       this.loading = false
+      this.$emit('process', false)
```

Comment:
if we add the above finally emit, we don't need this line

Reply:
done